src/test/java/com/google/javascript/clutz/partial/import_without_reexport.js

```diff
@@ -0,0 +1,8 @@
+goog.module('import.without.reexport');
+
```

Comment:
Add a comment what this is testing. It is had to see why the import without reexport matters, without understanding current working of aliasMap.

Long-term I think we should just parse the 'export = ', it is too much of jump to assume that the last import is reexported.

Reply:
Added a comment.

Agreed that's the right solution.